orisintel/laravel-migration-snapshot
# Feature/after load

### Manual test steps
1. checkout locally
2. composer require local copy in other Laravel project
3. add `after-load` closure to its `config/migration-snapshot.php`
4. run `php artisan migrate:load`
5. verify `after-load` closure has run and console reads "Ran After-load"
